networkit/__init__.py

```diff
@@ -89,6 +89,11 @@
 except ImportError:
 	raise MissingDependencyError("viztasks")
 
+try:
+	from . import csbridge
+except ImportError:
+	raise MissingDependencyError("csbridge")
```

Comment:
`csbridge` should not be available by default. (I consider it an anti-pattern that the existing modules are all available by default, but that cannot be changed without breaking lots of user code.)

Reply:
Ok, removed the import.